docs/src/main/sphinx/release/release-382.md

```diff
@@ -0,0 +1,58 @@
+# Release 382 (25 May 2022)
+
+## General
+
+* Add support for fault-tolerant execution with [exchange spooling on Google Cloud Storage](fte-exchange-manager). ({issue}`12360`)
+* Drop support for exchange spooling on S3 with for the legacy schemes `s3n://` and `s3a://`. ({issue}`12360`)
+* Improve join performance. ({issue}`12257`)
+* Improve coordinator memory management for task-based fault-tolerant execution.
+  This decreases the chance of queries failing with the
+  `EXCEEDED_TASK_DESCRIPTOR_STORAGE_CAPACITY` error code. ({issue}`12478`)
+
+## BigQuery connector
+
+* Add support for [using cached query results of BigQuery](https://cloud.google.com/bigquery/docs/cached-results).
+  This can be enabled using the `bigquery.query-results-cache.enabled` configuration property. ({issue}`12408`)
+* Support reading wildcard tables. ({issue}`4124`)
+
+## Delta Lake connector
+
+* Support predicate pushdown for `timestamp with time zone` type. ({issue}`12007`)
+* Add support for adding columns with `ALTER TABLE`. ({issue}`12371`)
+
+## Hive connector
+
+* Allow metastore caching without caching partition-level information. This can
```

Comment:
This is confusing. What does "caching without caching partition-level information" mean?